src/test/java/HomeWork1/SharelaneTests.java

```diff
@@ -0,0 +1,193 @@
+package HomeWork1;
+
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+import org.openqa.selenium.By;
+
+public class SharelaneTests {
+    @Test
+    public void signUpPositiveTest1() {
```

Comment:
Do not use numers in method names like test1, test2. Please better add more datailed name. It's OK for test methods to have long names.